Mathlib/RingTheory/Valuation/ValuativeRel/Basic.lean

```diff
@@ -806,6 +821,10 @@ lemma mapValueGroupWithZero_valuation (a : A) :
 
 end ValuativeExtension
 
+instance {R : Type*} [CommRing R] [ValuativeRel R] (S : Subring R) :
+    ValuativeExtension S R where
+  rel_iff_rel _ _ := .rfl
```

Comment:
Is this generally true for any `.comap`?

Reply:
Yes but the general comap one couldn't be its own instance.

Reply:
Makes sense -- provide it as a lemma?